source/addComment.js

```diff
@@ -1,3 +1,47 @@
-$(document).ready(function(){
+$(document).ready(function() {
   // add your code here
-})
+  const new_comment_button = document.querySelector('#new_comment_button');
+  new_comment_button.addEventListener('click', showCommentForm);
+
+  const form = document.querySelector('#comment_form');
+  form.addEventListener('submit', evt => {
+    evt.preventDefault();
+    handleComment();
+    hideCommentForm();
+  });
+});
+
+
+const hideCommentForm = () => {
+  const form = document.querySelector('#comment_form');
+  form.style.display = 'none';
+};
+
+const showCommentForm = () => {
```

Comment:
You might be able to DRY up these two functions by passing the css selector and the display value